projects/extension/src/background/AppMediator.ts

```diff
@@ -69,6 +75,30 @@ export class AppMediator {
   }
 
   processSmoldotMessage(message: JsonRpcResponse): boolean {
+    if (this.#state === 'disconnected') {
+      // Shouldn't happen - we remove the AppMediator from the smoldot's apps
+      // when we disconnect (below).
+      console.warn(`Asked a disconnected UApp (${this.name}) to process a message from ${this.#smoldotName as string}`);
```

Comment:
respective info message should return

Reply:
what do you mean?  I should throw it as an error?  Even though this state shouldn't happen I was being cautious that it might in an unconsidered scenario / race-condition hence logging a warning.

Reply:
We can't send the message back at this point .. the port is disconnected.  The scenario i have in mind is that the UApp sends an RPC message immediately before disconnecting... we might get the response back for that message and already be disconnected (although we shouldn't because unregistered the app with the smoldot).  If we start seeing these warnings in the dev console there is probably a bug in the AppMediator code

Reply:
So maybe `console.error` is better?  It shows up as an error but doesn't cause the extension to crash (the extension may be servicing otherwise working UApps)

Reply:
I agree. Its better to "save" the flow at this point and just inform through console

Reply:
OK done